src/opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1391,6 +1391,19 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
             new_child_group.updateParent(parent_name);
             this->snapshots.back().groups.update( std::move(new_child_group) );
         }
+
+        // Update network if required
+        auto network = this->snapshots.back().network.get();
```

Comment:
Although not totally sure about the scenario here, while I **assume** this only applies to Standard Network, right?  It looks like when we add a group hierarchy, the network will be updated accordingly?

Reply:
Oops, it **should** only apply for standard network, but as it stands now it could inadvertently happen also for extended. Will fix, thanks!!